frontend/ui/impl/src/test/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorReportSubmitterTest.java

```diff
@@ -0,0 +1,49 @@
+package com.virtuslab.gitmachete.frontend.ui.impl;
+
+import static org.mockito.ArgumentMatchers.*;
+
+import java.util.HashMap;
+
+import com.intellij.diagnostic.AbstractMessage;
+import com.intellij.diagnostic.IdeaReportingEvent;
+import com.intellij.diagnostic.LogMessage;
+import com.intellij.openapi.diagnostic.IdeaLoggingEvent;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({GitMacheteErrorReportSubmitter.class})
+public class GitMacheteErrorReportSubmitterTest {
+  @Test
+  public void shouldNotThrowWhenGettingBugTemplate() throws Exception {
+    GitMacheteErrorReportSubmitter reportSubmitter = new GitMacheteErrorReportSubmitter();
+
+    Whitebox.invokeMethod(reportSubmitter, "getBugTemplate");
```

Comment:
Hmm for clarity... how would that differ from a direct `reportSubmitter.getBugTemplate()`?

Reply:
Oh okay, private method